packages/xo-web/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "name": "xo-web",
-  "version": "5.50.2",
+  "version": "5.51.0",
```

Comment:
We're doing a patch release, I think it makes sense to use a patch version here too.